docs/index.rst

```diff
@@ -2,9 +2,12 @@
 
 .. _home:
 
+.. class:: vale-ignore
```

Comment:
Looks like it's the wrong way... 
I opened a PR to add yarf to the accepted word list: https://github.com/canonical/documentation-style-guide/pull/136

Reply:
```suggestion
```
The above PR has been merged, this line can be removed. I was advised to escape the parentheses with Vale token exceptions, but the current title still passes the check without exception rule, so I suppose nothing else is needed